.github/workflows/test_and_publish_charm.yaml

```diff
@@ -27,7 +27,7 @@ on:
       integration-test-provider-channel:
         description: Actions operator provider channel for the integration tests as per https://github.com/charmed-kubernetes/actions-operator#usage
         type: string
-        default: 1.26/stable
+        default: 5.12/stable
```

Comment:
Hmm I don't understand this 5.12, you're propagating latest everywhere unless here

Reply:
Fixed